core/vm/opcodes.go

```diff
@@ -254,7 +254,7 @@ var opCodeToString = map[OpCode]string{
 	MULMOD: "MULMOD",
 
 	// 0x20 range - crypto.
-	SHA3: "SHA3",
+	KECCAK256: "KECCAK256",
```

Comment:
This will 'break' the API for differential tracing. It's pretty easy to fix in the tracer shims though, so not a blocker for me

Reply:
If you have some links, I can try submitting support pre-emptively. Note that evmone already outputs `KECCAK256` so the shims should be updated to accept both in any case.

Reply:
I think e.g. this just need an alias: https://github.com/holiman/goevmlab/blob/0d79a9a3f8e4f25dcac009f1e4b7f0e1d458c238/ops/operations.go#L304 . It's not a biggie, I can fix whenever I need it next

Reply:
Here's my attempt: https://github.com/holiman/goevmlab/pull/46